### DISCUSSION TITLE
Clicking File or Directory in torrent creation window causes crash

### DISCUSSION THREAD

**Discussion by rflois** [openscopeproject/TrguiNG#58] [General]
I'm running 0.8.1 on Mac. When I click the File or Directory buttons in the torrent creation window, the app immediately crashes. I do have path mappings enabled that work fine within the app when choosing Open from the contextual menu.

**Comment by qu1ck** [comment]
Fixed in [9761164](https://github.com/openscopeproject/TrguiNG/commit/97611641b7856a6e56cf3fc2d214076473ab065e)

**Reply by rflois** [reply]
Confirmed. Thanks!
